### PR TITLE
feat: allow mp4 uploads to storymaps

### PR DESCRIPTION
**Pull request by jodiezhuu**
## Description
Add a media validation check to handle_config_media to 

### Related PRs
https://github.com/techmatters/terraso-backend/pull/606

### Related Issues
https://github.com/techmatters/terraso-product/issues/145

### Verification steps
1. On homepage, click on "Create New Story Map"
2. Add a title and click on "Add new chapter"
3. Click on "Add Media" and upload a mp4 file
4. Click "Publish" and should be successful

### CODE REVIEW

File: src/storyMap/components/StoryMapForm/EditableMedia.js [hunk 1]
```diff
@@ -441,6 +441,79 @@ const EditableAudio = props => {
...
+    }
+    if (video.contentId) {
+      return getMediaFile(video.contentId)
+    }
+    return null;
+  }, [video, getMediaFile]);
+
+  return (
+    <Stack>
+      <video key={id} style={{ width: '100%' }} controls>
```
Comment (josebui) [inline src/storyMap/components/StoryMapForm/EditableMedia.js:466]: Issue: It seems you need to add the same omit we have for audio: {/* eslint-disable-next-line jsx-a11y/media-has-caption */}

File: src/storyMap/components/StoryMapForm/EditableMedia.js [hunk 2]
```diff
@@ -441,6 +441,80 @@ const EditableAudio = props => {
...
+          onConfirm={onDelete}
+          loading={processing}
+          variant="text"
+          buttonProps={{
+            title: t('storyMap.form_media_delete_label'),
+            sx: {
+              minWidth: 'auto',
+            },
+          }}
+          confirmTitle={t('storyMap.form_media_video_delete_confirm_title')}
```
Comment (josebui) [inline src/storyMap/components/StoryMapForm/EditableMedia.js:507]: Question: Are these string sin the en_us file?

Comment (jodiezhuu) [reply src/storyMap/components/StoryMapForm/EditableMedia.js:507]: Ya, it was part of another ticket that was already merged in main

**Review by josebui — APPROVED** [review]

**Comment by josebui** [comment]
@jodiezhuu please before merging push it to staging so that Hana and Derek can test it first

**Pull request merged**